### ISSUE TITLE
Crash with peer time out in DHT

### ISSUE THREAD

**Issue by xoriole** [Tribler/tribler#3957]
Tribler crashed with peer time out runtime error in DHT
```python
Traceback (most recent call last):
File "TriblerGUI\event_request_manager.py", line 130, in on_read_data
RuntimeError: [Failure instance: Traceback (failure with no frames): <type 'exceptions.RuntimeError'>: Node Peer<{IP}:7759, iIfgmKR5QBqZeRTUTghJK4KARLY=> timeout
]
```

**Comment by Dmole** [comment]
Same, just idling;
```
Traceback (most recent call last):
  File "/usr/share/tribler/TriblerGUI/event_request_manager.py", line 130, in on_read_data
    raise RuntimeError(json_dict["event"]["text"])
RuntimeError: [Failure instance: Traceback (failure with no frames): <type 'exceptions.RuntimeError'>: Node Peer<$IP:35010, Vtpg/5j8OuI1Izm0CYPVoAP/6sQ=> timeout
]
```

**Comment by devos50** [comment]
I suspect that we did not implement a proper errback somewhere. @egbertbouman could you take a look at this?

**Comment by Dmole** [comment]
@devos50 Please make a new build when this is fixed because it crashes v7.1.0 reliably every few hours (<12h).

**Comment by qstokkink** [comment]
This should've been fixed by #3969.
